layers/poi/layer.sql

```diff
@@ -105,25 +134,7 @@ AS $$
             WHERE zoom_level >= 14
                 AND (name <> '' OR (subclass <> 'garden' AND subclass <> 'park'))
         ) as poi_union
-    WHERE
-        subclass NOT IN ('yes', 'Yes', 'no', 'No', 'none')
```

Comment:
The overall form of the new CASE/WHEN looks more readable to me, thanks :)
But didn't we lose this general case? What about when `mapping_key` is not `shop` nor `office`? Seems to me it should be put in place of the final `ELSE TRUE` in your new implementation, to address possible other cases.

Reply:
For the case of offices, we thought that for `office=yes` can be relevant (eg. https://www.openstreetmap.org/way/116137151), that's the reason why I made the previous implementation a bit harder to read by adding a list of exception to this almost global filter.

> But didn't we lose this general case? What about when `mapping_key` is not `shop` nor `office`? Seems to me it should be put in place of the final `ELSE TRUE` in your new implementation, to address possible other cases.

Yes nice catch, I'll fix this in the instant.
Note that this `ELSE` branch would currently apply to craft, sport and tourism. It makes me wonder if we really don't wan't to import `tourism=yes`, but this can be for another time.